chia/util/api_decorators.py

```diff
@@ -71,6 +72,7 @@ def wrapper(self: Self, original: Union[bytes, S], *args: P.args, **kwargs: P.kw
         message_name_bytes = f"{message_name}_bytes"
 
         metadata = ApiMetadata(
+            endpoint=f.__name__,
```

Comment:
I tried to somehow get the name of the decorated methods passed into `call_api`'s `request_method` parameter but i couldn't find a reasonable way so i ended up with `ApiMetadata.endpoint`. We get the wrapped method there so a direct `__name__` access would only result in `wrapper` as name since we get the below:

```
<function chia.util.api_decorators.api_request.<locals>.inner.<locals>.wrapper(self: 'Self', original: 'Union[bytes, S]', *args: 'P.args', **kwargs: 'P.kwargs') -> 'R'>
```